src/patternfly/components/Table/table.scss

```diff
@@ -193,57 +216,85 @@
   --#{$table}--m-compact__action--PaddingTop: var(--pf-t--global--spacer--xs);
   --#{$table}--m-compact__action--PaddingBottom: var(--pf-t--global--spacer--xs);
 
+  // * Table expandable tbody
+  --#{$table}--m-expandable__tbody--BorderBottomWidth: var(--pf-t--global--border--width--divider--default);
+  --#{$table}--m-expandable__tbody--BorderBottomColor: var(--pf-t--global--border--color--default);
+
   // * Table expandable row expanded
   --#{$table}__expandable-row--m-expanded--BorderBottomColor: var(--pf-t--global--border--color--default);
 
   // * Table tr clickable
-  --#{$table}__tr--m-clickable--BackgroundColor: transparent;
+  --#{$table}__tr--m-clickable--BackgroundColor: var(--pf-t--global--background--color--primary--default);
```

Comment:
Curious why this can't be transparent? The table has a background color that seems to show.

Reply:
Updated